Desktop: Fixes: #4612: Fix mermaid diagrams in WYSIWYG editor

mablin7 opened the pull request:
The white space in the mermaid markup is removed during parsing by TinyMCE if it's not in one of [these](https://github.com/tinymce/tinymce/blob/ad61fbbe14ce58174ef7c8635c368696c61c18fd/modules/tinymce/src/core/main/ts/api/html/Schema.ts#L471) tags and that caused mermaid to throw a syntax error.

Simply changing the wrapper tag from `div` to `pre` preserves all the newlines, without affecting any other functionality.

Here's the split view (unchanged by this PR):
![split-view](https://user-images.githubusercontent.com/7415668/111182685-71cd8f00-85af-11eb-9f53-52e8380c806c.png)

Here's the broken WYSIWYG view:
![wysiwyg-broken](https://user-images.githubusercontent.com/7415668/111182679-709c6200-85af-11eb-887d-08b60f77e5de.png)

And here's the fixed diagram:
![wysiwyg-fixed](https://user-images.githubusercontent.com/7415668/111182682-7134f880-85af-11eb-982a-fd1ad978246e.png)

laurent22 commented:
Indeed that makes sense. Do you have a screenshot of how a diagram looks before and after this change?

Also could you please add a short comment explaining why PRE is required there (since we can't unit test and it's not obvious it has anything to do with TinyMCE)? Also please add a link to this pull request.

mablin7 commented:
Hey! I updated the PR with the screenshots. I'll add the comments right away, just a quick question: would prefer if I squashed that and the original commit into one?

laurent22 commented:
Perfect, thanks for adding the comment and screenshots! No need to squash the commits as I do it when I merge the pull request.
